frontend/src/components/pages/Account.js

```diff
@@ -121,6 +128,21 @@ export function Account() {
                         </form>
                     </section>
                 );
+            case 'signOutAccount':
+                return (
+                    <section>
+                        <h1 className="text-2xl font-bold mb-4">Signout Of Account</h1>
+                        <p>Click the button below to signout <br />    
+                        </p>
+                        <button
```

Comment:
nit: should add a little more space between this button and the text above. Just ad ``mt-2`` to the button className bellow.